tests/test_kratos_solver_io.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from stem.IO.kratos_solver_io import KratosSolverIO
+from stem.solver import *
+from stem.model_part import ModelPart, BodyModelPart
+
+from tests.utils import TestUtils
+
+
+class TestKratosSolverIO:
+
+    def test_create_settings_dictionary(self):
```

Comment:
missing all the possible combinations of solver settings 
what is the code coverage of solver?

Reply:
to be tested separetly, where it is check if kratos actually runs well

#50